src/packages/core/property-editor/uis/tree-picker/config/source-type-picker/property-editor-ui-tree-picker-source-type-picker.element.ts

```diff
@@ -9,6 +9,8 @@ import { UmbLitElement } from '@umbraco-cms/internal/lit-element';
 import { UMB_PROPERTY_DATASET_CONTEXT } from '@umbraco-cms/backoffice/property';
 import { UmbPropertyValueChangeEvent } from '@umbraco-cms/backoffice/property-editor';
 
+import '../../../../../../documents/document-types/components/input-document-type/input-document-type.element.js';
```

Comment:
Consider if this could be imported from `@umbraco-cms/backoffice/components`. Perhaps it's missing an export.

Reply:
I believe there was another recent change that caused me to do this, but just re-tested it and it works without the `import`, so I've removed it now.